sdc/tests/test_series.py

```diff
@@ -1482,7 +1482,7 @@ def test_series_op5(self):
 
     @skip_parallel
     def test_series_op5_integer_scalar(self):
-        arithmetic_methods = ('add', 'sub', 'mul', 'div', 'truediv', 'floordiv', 'mod', 'pow')
+        arithmetic_methods = ('sub', 'mul', 'div', 'truediv', 'floordiv', 'mod', 'pow')
```

Comment:
I don't think you should remove this function from the test (this use case is still valid), instead you can add a check_dtype=False param to assert_series_equal function and add a comment to state why it's used.